02-dev_apps/first-actor/manifest.md

```diff
@@ -0,0 +1,32 @@
+As we learn in the introduction to `wasmcloud`, we can bootstrap the runtime with a manifest file. Here, we are going to do a quick review of the file.
+
+<details>
+  <summary>Rust</summary>
+  
+  `rust/manifest.yaml`{{open}}
+  
+</details>
+<details>
+  <summary>Go</summary>
+
+`go/manifest.yaml`{{open}}
+
+</details>
+<details>
+  <summary>Assembly Script</summary>
+
+`assemblyscript/manifest.yaml`{{open}}
+
+</details>
+
+It is important to notice that `actor` fields in the link has an ID provided. That ID will NOT be the same for you on your system. We have two options:
+
+- You can use `wash` to give you the actor ID of your actor
+  `wash claims inspect localhost:5000/calc:0.1.0 --insecure`{{execute}}
+- You can use the environmental variable to override the current ID
+  `export CLIENT_ACTOR=$(wash claims inspect localhost:5000/calc:0.1.0 --insecure -o json | jq -r '.module')`{{execute}}
```

Comment:
change this to `wash claims inspect` with the signed wasm module (you may need to put this in a polyglot section, apologies)